styleguide/src/utils/elapsedDuration.ts

```diff
@@ -0,0 +1,26 @@
+const MS_IN_SECOND = 1000
+const SECONDS_IN_MINUTE = 60
+const SECONDS_IN_HOUR = 3600
+
+export function elapsedDuration(createdAt: string, finishedAt: string): string {
```

Comment:
Can we have a jsdoc here explaining what format the strings in the `createdAt` and `finishedAt` parameters should be? And what format the string returned by `elapsedDuration` is?

Reply:
Sorry missed this comment. Added in https://github.com/smartcontractkit/chainlink/pull/2140/commits/2a38af210978ee9b4ac2dbccba757631360a8a2c. Is that along the lines of what you were thinking?